Makefile

```diff
@@ -147,7 +147,7 @@ test: \
 .PHONY: test
 
 test-back: ## run back-end tests
-	bin/pytest
+	bin/pytest -k test_api_course_run_read_detail_not_listed
```

Comment:
debug fail

why don't you use the command `bin/pytest` instead of `make test-back`?

Reply:
Just bad habit! Indeed I'll update my test command as I make this mistake too often.

<img width="133" alt="Capture d’écran 2022-10-24 à 09 12 14" src="https://user-images.githubusercontent.com/9265241/197468207-6eb6c548-319e-4f1a-9bbd-432221ee97e9.png">